producer.py

```diff
@@ -46,14 +43,23 @@
 from thoth.python import AIOSource
 from thoth.python.exceptions import NotFoundError
 
+try:
+    __version__ = version(__package__)
+except PackageNotFoundError:
+    __version__ = "v0.11.10"  # TODO makes this unnecessary by fixing users of the package
```

Comment:
Have this default to `dev`? I think local development is the most likely cause of version not being found.